deepchecks/utils/abstracts/confusion_matrix_abstract.py

```diff
@@ -77,3 +79,77 @@ def create_confusion_matrix_figure(confusion_matrix_data: np.ndarray, classes_na
     fig.update_yaxes(title='True value', type='category', constrain='domain', autorange='reversed')
 
     return fig
+
+
+def misclassified_samples_lower_than_condition(value: np.ndarray,
+                                               misclassified_samples_threshold: float) -> ConditionResult:
+    """Condition function that checks if the misclassified samples in the confusion matrix is below threshold.
+
+    Parameters
+    ----------
+    value: np.ndarray
+        Value of the confusion matrix
+    misclassified_samples_threshold: float
+        Ratio of samples to be used for comparison in the condition (Value should be between 0 - 1 inclusive)
+
+    Raises
+    ------
+    ValidationError
+        if the value of `misclassified_samples_threshold` parameter is not between 0 - 1 inclusive.
+
+    Returns
+    -------
+    ConditionResult
+        - ConditionCategory.PASS, if all the misclassified samples in the confusion
+        matrix are less than `misclassified_samples_threshold` ratio
+        - ConditionCategory.FAIL, if the misclassified samples in the confusion matrix
+        are more than the `misclassified_samples_threshold` ratio
+    """
+    if misclassified_samples_threshold < 0 or misclassified_samples_threshold > 1:
+        raise ValidationError(
+           'Condition requires the parameter "misclassified_samples_threshold" '
+           f'to be between 0 and 1 inclusive but got {misclassified_samples_threshold}'
+        )
+
+    # Computing the total number of samples from the confusion matrix
+    total_samples = np.sum(value)
+
+    # Number of threshold samples based on the 'misclassified_samples_threshold' parameter
+    thresh_samples = round(np.ceil(misclassified_samples_threshold * total_samples))
+
+    # m is the number of rows in the confusion matrix and
+    # n is the number of columns in the confusion matrix
+    m, n = value.shape[0], value.shape[1]
+
+    # Variables to keep track of cells above 'thresh_samples'
+    n_cells_above_thresh = 0
+    max_samples_in_cell_above_thresh = thresh_samples
```

Comment:
keep info only on which cell is it - see comment of error message

Reply:
Now, I keep track of the row and col indices of the max samples cell in the confusion matrix which is above the threshold.